generator/list.tmpl

```diff
@@ -2,14 +2,14 @@
 # SIGs and Working Groups
 
 Most community activity is organized into Special Interest Groups (SIGs),
-time bounded Working Groups, and the [community meeting](communication.md#weekly-meeting).
+time bounded Working Groups, and the [community meeting](communication/README.md#weekly-meeting).
 
 SIGs follow these [guidelines](governance.md) although each of these groups may operate a little differently
 depending on their needs and workflow.
 
 Each group's material is in its subdirectory in this project.
 
-When the need arises, a [new SIG can be created](sig-creation-procedure.md)
+When the need arises, a [new SIG can be created](sig-governance.md#sig-creation-and-maintenance-procedure.md).
```

Comment:
This anchor looks bad.

Reply:
> This anchor looks bad.

Good catch.  Fixed with ca74dc32 -> 4833c3c1.